frontend/tests/features/wheresMyGene/wheresMyGene.test.ts

```diff
@@ -354,12 +349,14 @@ describe("Where's My Gene", () => {
         testId: COMPARE_DROPDOWN_ID,
       });
 
+      await waitForElementToBeRemoved(page, "loading-spinner");
```

Comment:
loading the compare data was taking a while, so we're waiting for the load to finish here

Reply:
let's at this comment in the code to persist this intention!